lib/src/conflicts.rs

```diff
@@ -355,6 +338,33 @@ fn materialize_conflict_hunks(
     Ok(())
 }
 
+/// Write a positive snapshot (side) of a conflict
+fn write_side(add_index: usize, data: &[u8], output: &mut dyn Write) -> std::io::Result<()> {
+    output.write_all(CONFLICT_PLUS_LINE)?;
+    output.write_all(format!(" Contents of side #{}\n", add_index + 1).as_bytes())?;
```

Comment:
Yeah, that makes things a lot easier to read actually, thanks! I had to make it be `&str` because I couldn't find a way to construct a `&BStr` in a `const` context and I didn't want to use a `LazyLock`.